### PR TITLE
Initial support for Debian 12 Bookworm

### PR DESCRIPTION
**Pull request by ReillyBrogan**
**What this PR does / why we need it**:
Initial support for the upcoming Debian 12

**Special notes for your reviewer**:
This currently requires a custom image to be built. That image will need to have the debian package base-files 12.4 installed which is not yet available in testing (but is available in unstable and is expected to be migrated soon).

### CODE REVIEW

File: docs/operations/images.md [hunk 1]
```diff
@@ -123,6 +123,12 @@ az vm image list --all --output table \
   --publisher Debian --offer debian-11 --sku 11-gen2
 ```
 
+### Debian 12 (Bookworm)
+
+Debian 12 is based on Kernel version **6.1** which has no known major Kernel bugs and fully supports all Cilium features.
+
+
```
Comment (hakman) [inline docs/operations/images.md:130]: Please add a link to this in the Matrix.
```suggestion
At the moment there is no official image published.
```

**Review by hakman — CHANGES_REQUESTED** [review]

File: docs/operations/images.md (hunk 1 above)
Comment (ReillyBrogan) [reply docs/operations/images.md:130]: What version should I put as the starting version? `1.27`? `1.26.3`?

Comment (hakman) [reply docs/operations/images.md:130]: I would go with 1.26.3.

**Comment by ReillyBrogan** [comment]
@hakman I was actually going to mention you anyway. I see that there's a check for Debian 11 in `sysctls.go` regarding a Router Advertisement sysctl. I read through all of the PRs/issue I found on it and wasn't able to figure out the exact reason for this distro-specific workaround. 

My best guess is that this is because Debian 11 does not use systemd-networkd while having a new-ish kernel, is this workaround likely to be required for Debian 12 as well if it does not use systemd-networkd? I haven't yet created a "fresh" Debian 12 install to see what it uses by default...

**Comment by hakman** [comment]
This setting is needed only for IPv6, where addresses are not allocated without it (due to forwarding being enabled).
For `systemd-networkd`, there's no need to do such things:
https://www.freedesktop.org/software/systemd/man/systemd.network.html#IPv6AcceptRA=

**Review by hakman — APPROVED** [review]

**Comment by ReillyBrogan** [comment]
I guess we'll just wait and see if the official Bookworm images ship with systemd-networkd enabled or not.

**Pull request merged**